app/src/androidTest/java/nextstep/signup/component/PasswordConfirmTextFieldTest.kt

```diff
@@ -0,0 +1,102 @@
+package nextstep.signup.component
+
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.isDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onAllNodesWithText
+import androidx.compose.ui.test.onFirst
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performTextInput
+import nextstep.signup.validator.SignupInfoValidator
+import org.junit.Rule
+import org.junit.Test
+
+class PasswordConfirmTextFieldTest {
+
+    @get:Rule
+    val composeTestRule = createComposeRule()
+
+
+    @Test
+    fun `비밀번호_확인_라벨이_보여진다`() {
+        // given
+        val label = "Password Confirm"
+
+        // when
+        composeTestRule.setContent {
+            PasswordConfirmTextFiled(
+                text = "passwordConfirm",
+                validator = SignupInfoValidator.PasswordConfirm { "passwordConfirm" },
+                onValueChange = { },
+            )
+        }
+
+        // then
+        composeTestRule
+            .onNodeWithText(label)
+            .isDisplayed()
+    }
+
+    @Test
+    fun `비밀번호_확인은_평문으로_보이지_않는다`() {
+        composeTestRule.setContent {
+            PasswordConfirmTextFiled(
+                text = "1q2w3e4r",
+                validator = SignupInfoValidator.PasswordConfirm { "1q2w3e4r" },
+                onValueChange = { },
+            )
+        }
+
+        // then
+        composeTestRule
+            .onAllNodesWithText("\u2022\u2022\u2022\u2022\u2022\u2022\u2022\u2022")
+            .onFirst()
+            .assertIsDisplayed()
```

Comment:
유니코드 값을 넣어도 좋지만, 눈에 보이는 값을 넣어주는 건 어떨까요?
유니코드 값을 넣기를 희망하신다면, 테스트 코드를 보는 다른 사람이 한 눈에 이해하기 쉽게 주석으로 보이는 값을 적어보셔도 좋겠어요!

```suggestion
        composeTestRule
            .onAllNodesWithText("••••••••")
            .onFirst()
            .assertIsDisplayed()
```

Reply:
다른사람들이 보면 어떤 실제 어떻게 보이는지 알 수가 없는데 이 특수문자는 이런 유니코드야! 라는 생각에만 매몰된 것 같네요.. 감사합니다!